pizzalab/pizzalabapp/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+
+from pizzalabapp.models import Pizza, Ingredient, Order, OrderItem
+
+
+class IngredientSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = '__all__'
+
+
+class IngredientListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = ('name',)
+
+
+class PizzaSerializer(serializers.HyperlinkedModelSerializer):
+    ingredients = IngredientListSerializer(many=True)
+    has_allergen_ingredients = serializers.SerializerMethodField()
+    has_lactose_ingredients = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Pizza
+        fields = '__all__'
+
+    def get_has_allergen_ingredients(self, instance):
+        for ingredient in instance.ingredients.all():
+            if ingredient.is_allergen:
+                return True
+        return False
```

Comment:
Add blank line before return statement - not a PEP must-do but helps with easier understanding of the code, emphasizes what you expect from the method/function